getting-started/traced-example/package.json

```diff
@@ -9,13 +9,13 @@
   "author": "OpenTelemetry Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@opentelemetry/core": "0.23.0",
-    "@opentelemetry/exporter-zipkin": "0.23.0",
-    "@opentelemetry/instrumentation-express": "^0.23.0",
-    "@opentelemetry/instrumentation-http": "0.23.0",
-    "@opentelemetry/instrumentation": "0.23.0",
-    "@opentelemetry/node": "0.23.0",
-    "@opentelemetry/tracing": "0.23.0",
+    "@opentelemetry/core": "0.24.0",
+    "@opentelemetry/exporter-zipkin": "0.24.0",
+    "@opentelemetry/instrumentation-express": "^0.24.0",
```

Comment:
This one doesn't exist yet.
I think we should remove the sample using express instrumentation from this repo.

Reply:
> This one doesn't exist yet.

Good catch

> I think we should remove the sample using express instrumentation from this repo.

I agree. Using the http instrumentation is probably good enough for the getting-started guide. FWIW, getting-started is probably going to be removed entirely now that we have more updated website docs.